sale_configurator_option/models/product.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2020 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# @author Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.addons import decimal_precision as dp
+
+
+class ProductConfiguratorOption(models.Model):
+    _name = 'product.configurator.option'
+    _order = "sequence, id"
+    _rec_name = "product_id"
+    _description = 'Product Configurator Option'
+
+    def _get_default_product_uom_id(self):
+        return self.env['uom.uom'].search([], limit=1, order='id').id
+
+    product_tmpl_id = fields.Many2one(
+        'product.template', 'Product Template',
+        auto_join=True, index=True, ondelete="cascade", required=True)
+    product_id = fields.Many2one(
+        'product.product', 'Option', required=True)
+    product_uom = fields.Many2one(
```

Comment:
please keep product_uom_id
In order to be consistente with mrp.bom.line : https://github.com/odoo/odoo/blob/13.0/addons/mrp/models/mrp_bom.py#L265